soccer/planning/tests/CreatePathTest.cpp

```diff
@@ -0,0 +1,3 @@
+#include "planning/low_level/CreatePath.hpp"
+
+#include <gtest/gtest.h>
```

Comment:
Is this intentional?

Reply:
Yes? What about it?

Reply:
It was just an empty file with two includes

Reply:
Oh rip, thought I actually put stuff in here. Honestly I think as much as can be tested with this is covered by other tests, I'll delete it.